dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ElasticsearchUtil.java

```diff
@@ -119,18 +121,46 @@ public static void setLiveAndWorkingIndicesToWriteMode() throws ElasticsearchRes
      * @return boolean
      */
     public static boolean isClusterInReadOnlyMode(){
-        try {
-            final ClusterGetSettingsResponse response = RestHighLevelClientProvider.getInstance()
-                    .getClient().cluster()
-                    .getSettings(new ClusterGetSettingsRequest(), RequestOptions.DEFAULT);
+        final ClusterGetSettingsResponse response = getClusterSettings();
+
+        return Boolean.parseBoolean(response.getSetting("cluster.blocks.read_only"))
+                || Boolean.parseBoolean(response.getSetting("cluster.blocks.read_only_allow_delete"));
+    }
+
+    /**
+     * Return how often Elasticsearch check on disk usage for each node in the cluster
+     * @return boolean
+     */
+    public static long getClusterUpdateInterval(){
+        final ClusterGetSettingsResponse response = getClusterSettings();
+        final String intervalString = response.getSetting("cluster.info.update.interval");
+        final long intervalInMillis = getIntervalInMillis(intervalString == null ? ES_DEFAULT_VALUE : intervalString);
+
+        return intervalInMillis;
```

Comment:
Codacy found an issue: [Consider simply returning the value vs storing it in local variable 'intervalInMillis'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5714647)